sdk/appconfiguration/azure-data-appconfiguration/src/main/java/com/azure/data/appconfiguration/ConfigurationClient.java

```diff
@@ -500,11 +502,12 @@
      *
```

Comment:
Relevant code samples should be extracted out into snippets files and updated to latest API.